web_src/css/markup/content.css

```diff
@@ -3,6 +3,7 @@
   font-size: 16px;
   line-height: 1.5 !important;
   overflow-wrap: break-word;
+  word-break: break-all;
```

Comment:
But `break-all` makes English difficult to read

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Reply:
It will indeed be like this. Then let's find a compromise. How about limiting the width of the container and allowing the content to scroll inside it? This way, we can prevent the content from overflowing the page and disrupting the layout. Is that okay?